server/utils/nunjucksSetup.ts

```diff
@@ -66,4 +67,6 @@ export default function nunjucksSetup(app: express.Express, applicationInfo: App
   njkEnv.addFilter('formatDate', formatDate)
   njkEnv.addFilter('formatDateTime', formatDateTime)
   njkEnv.addFilter('formatTime', formatTime)
+  njkEnv.addFilter('nameCase', convertToTitleCase)
```

Comment:
We have `formatName` and `formatNamePart` in the profile, which works better than `convertToTitleCase` for people's names, as it also allows for apostrophes and spaces as well as dashes.
Another thing where having a shared library would be useful to avoid duplication.

Reply:
Have updated